src/cs.unittests.alox/Tests_Tutorial.cs(293):Tut_Exception() [00:00.000 +034 Âµs] [TestRunnerThread(10)] [WRN] [MYDOM]: >> Logging an exception: 
                                                                                                                       >>   Type:       System.Exception
                                                                                                                       >>   Message:    TestException Message
                                                                                                                       >>   HResult:    0x80131500
                                                                                                                       >>   Inner:   
                                                                                                                       >>     Type:       System.Exception
                                                                                                                       >>     Message:    InnerException Message
                                                                                                                       >>     HResult:    0x80131500
                                                                                                                       >>     Inner:   
                                                                                                                       >>       Type:       System.Exception
                                                                                                                       >>       Message:    Inner, inner Exception
                                                                                                                       >>       HResult:    0x80131500